apps/api/package.json

```diff
@@ -17,10 +17,6 @@
     "test": "vitest run --project unit",
     "test:watch": "vitest watch --project unit",
     "test:e2e": "vitest run --project e2e",
-    "test:e2e:watch": "vitest watch --project e2e",
```

Comment:
I removed these scripts:
- test:e2e:watch, there's no way anyone wants to restart teh e2e tests on every change imo.  If someone is using this, we can add it back.
- test:e2e:runner, not used anymore
- test:multi-chain and test:multi-chain:e2e, these are just shortcuts to running specific files, doesn't seem useful.